SECURITY.md

```diff
@@ -0,0 +1,21 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
```

Comment:
What versions are supported? @nohwnd is this ok?

Reply:
Do we actually need this? There never was a security fix for just Pester, and since I don't depend on anything external, apart from few lines of compiled c# code that does not have any dependencies itself there is very little opportunity for introducing and fixing a security bug. If there was a security bug it would likely be in powershell, and hence it would get fixed and delivered by powershell team.

Reply:
@asears polite nugde :)

Reply:
I can say that it's good practice and recommended by Github to have security.md, and an understanding of how to triage security vulnerabilities for OSS projects.  I don't see one in the Powershell repo.  Do they inherit their process from Microsoft? 

Up to you if it makes sense to include for Pester, another thing to maintain.  

I updated with some comments and could possibly create a security@pester.dev mailing list if that makes sense.
Or can delete the file and I'll publish the PR.

Here's the current vulnerabilities in .net framework, which Pester depends on.
https://www.cvedetails.com/vulnerability-list/vendor_id-26/product_id-2002/version_id-171205/Microsoft-.net-Framework-4.5.2.html

As part of the repo, Pester build framework depends indirectly on docker containers, chocolatey, nuget, Azure Devops, Powershell Gallery.  PesterTests depends on .NET.Test.Sdk, xunit.  

If there were credentials or some secrets inadvertently checked into the repo it would be good to have a channel to report them, and generally channels for security vulnerabilities are outside of Github issues.

Reply:
The other thing the security.md should do is enable the Security tab on the repo.

![image](https://user-images.githubusercontent.com/7253192/93453642-ebf2b480-f8a7-11ea-9439-25a2954a3c1b.png)

Reply:
Fair point. Added security@pester.dev, but you can keep both contacts.

Reply:
I also like how you rewrote it :)

Reply:
Thanks! Added to the security.md the email.  Is there a document on how to do a release / how to rollback a release that would need to be updated to incorporate the latest version changes / additions in security.md?  The maintenance part.

This might help automate as part of Devops, or could look for other solutions.
https://github.com/DavidWells/markdown-magic

Tagging the release might also work.

Reply:
I would just skip the version table altogether. Or said that 3.x.x is not actively maintained. 4.x.x is maintained, but only critical issues would be fixed. And 5.x.x is actively maintained. 

Patching previous specific versions would be borderline impossible, psgallery / nuget does not allow you to remove packages easily.